src/pynwb/__init__.py

```diff
@@ -186,7 +186,7 @@ def validate(**kwargs):
 
 class NWBHDF5IO(HDF5IO):
 
-    @docval({'name': 'path', 'type': str, 'doc': 'the path to the HDF5 file to write to'},
+    @docval({'name': 'path', 'type': str, 'doc': 'the path to the HDF5 file'},
             {'name': 'mode', 'type': str,
```

Comment:
this should be corrected to "to write to or read from"

Reply:
My intention was that the mode flag should describe the action and path should only describe the target.